scm/src/gmtb_scm.F90

```diff
@@ -130,6 +138,14 @@ subroutine gmtb_scm_main_sub()
                        physics%Init_parm, scm_state%n_cols, scm_state%lon,         &
                        scm_state%lat, scm_state%area)
   
+  !override radiation frequency
+  if (scm_state%force_rad_T > 0) then
+    !turn off radiation since it is already accounted for in the forcing
+    !set radiation calling periods to -1 in order to prevent lsswr/lslwr from being be set to true in GFS_time_vary_pre
+    physics%Model%nsswr = -1
```

Comment:
glad to see that this works

Reply:
Yes, thanks for the suggestion -- always nice to avoid new variables if possible.